views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/lib/xaudiojs/resampler.js

```diff
@@ -0,0 +1,174 @@
+"use strict";
```

Comment:
Also check that this file does not increase Eslint warning. If it does, please exclude it from eslint analysis.